pydata_sphinx_theme/__init__.py

```diff
@@ -250,11 +249,42 @@ def navbar_align_class():
             )
         return align_options[align]
 
+    def generate_google_analytics_script(kind, **kwargs):
+        """Handle the two types of google analytics id."""
+        id = kwargs.get("id")
+        if id:
+            if "G-" in id:
+                script = (
+                    f"<script async "
+                    "src='https://www.googletagmanager.com/gtag/js?id={id}'></script>\n"
+                    "<script>\n"
+                    "    window.dataLayer = window.dataLayer || [];\n"
+                    "    function gtag() { dataLayer.push(arguments);\n"
+                    f"    gtag('config', '{id}');\n"
+                    "</script>"
+                )
+            else:
+                script = (
+                    "<script async "
```

Comment:
wanna just make these f-strings? like:

```python
script = f"""
<script async ...>
  stuff
  {id}
  stuff
</script>
```

Reply:
Apologies for another push, I've missed this comment. That sounds good, the only issue is that the script will have a lot of white space on the final HTML page, but that should be alright